Lib/fontTools/cffLib/cff2mergePen.py

```diff
@@ -0,0 +1,303 @@
+# Copyright (c) 2009 Type Supply LLC
+
+from __future__ import print_function, division, absolute_import
+from fontTools.misc.fixedTools import otRound
+from fontTools.misc.psCharStrings import CFF2Subr
+from fontTools.pens.t2CharStringPen import T2CharStringPen
+from fontTools.cffLib.specializer import (commandsToProgram,
+										specializeCommands,
```

Comment:
unnecessary parens and extra space

Reply:
`from fontTools.cffLib.specializer import commandsToProgram, specializeCommands` is 78 chars long.